rust/src/lightning.rs

```diff
@@ -510,6 +527,14 @@ pub fn setup(
         IgnoringMessageHandler {},
     ));
 
+    // Step 17: Initialize routing ProbabilisticScorer
+    let scorer_path = format!("{ldk_data_dir}/scorer");
+    let scorer = Arc::new(Mutex::new(disk::read_scorer(
+        Path::new(&scorer_path),
+        Arc::clone(&network_graph),
+        Arc::clone(&logger),
+    )));
```

Comment:
❓ Why did we have to move this?

Reply:
to be able to start the background processor from the properties of the lightning system